tests/error/comprehension_errors/borrow_leaked2.err

```diff
@@ -0,0 +1,8 @@
+Error: Linearity violation (at $FILE:17:23)
+   | 
+15 | @guppy(module)
+16 | def foo(qs: list[qubit] @owned) -> list[int]:
+17 |     return [bar(q) for q in qs]
```

Comment:
obvs not for this PR but it would be super sweet if a hint said "try removing owned"

Reply:
Created #726